odoo/addons/base/security/ir.model.access.csv

```diff
@@ -65,12 +65,12 @@
 "access_res_partner_industry_group_user","res_partner_industry group_user","model_res_partner_industry","group_user",1,0,0,0
 "access_res_partner_industry_group_system","res_partner_industry group_system","model_res_partner_industry","group_system",1,1,1,1
 "access_res_partner_title_group_user","res_partner_title group_partner_manager","model_res_partner_title","group_partner_manager",1,1,1,1
-"access_res_partner_title_group_partner_manager","res_partner_title group_user","model_res_partner_title",,1,0,0,0
+"access_res_partner_title_group_partner_manager","res_partner_title group_user","model_res_partner_title","group_user",1,0,0,0
```

Comment:
Not sure why this should be restricted ... won't you have issues in public pages that display parner's title ? Disclosing that you use "Mr" and "Dr" does not seem a real security issue to me.

Reply:
Titles are so 1990. No security issue here but as the goal is to forbid `,,` rules, I remove the unncessary ones.

Reply:
German people use titles a lot. And those are generally my source of inspiration for everyday decisions.